ckan/lib/files/default/null.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
```

Comment:
This storage is not documented. What are it's use cases?

Reply:
just as memory storage, this one is intended for tests, that do not need to access the content of uploaded files. Forgot to document it